### DISCUSSION TITLE
arm v7编译的时候提示/usr/include/linux/errno.h:1:10: fatal error: asm/errno.h: No such file or directory

### DISCUSSION THREAD

**Discussion by xiedeacc** [1715173329/UnblockNeteaseMusic#751] [Q&A] [answered]
arm v7编译的时候提示/usr/include/linux/errno.h:1:10: fatal error: asm/errno.h: No such file or directory

**Comment by pan93412** [comment]
不是 UNM 的問題，大概是你忘記裝 linux-headers。

**Comment by xiedeacc** [comment]
不是这个问题，肯定不是没装Linux header，因为是交叉编译的，和宿主机没关系

**Reply by pan93412** [reply]
那也肯定不是 UNM 的問題⋯⋯你說的究竟是這個專案，還是 [UnblockNeteaseMusic/server-rust](https://github.com/UnblockNeteaseMusic/server-rust) 呢？前者很確定沒有需要用到 linux header 的部分。

**Comment by xiedeacc** [comment]
嗯 用rust肯定没这个问题，不过rust没有openwrt web配置界面

**Comment by xiedeacc** [comment, marked as answer]
看了下和unm确实没关系，unm依赖于nodejs，交叉编译的时候需要编译nodejs，nodejs过不了

**Comment by xiedeacc** [comment]
所以UNM确实没法支持armv7

**Reply by pan93412** [reply]
https://nodejs.org/zh-cn/download/current/ 絕對是支援，就是不知道你的跨編譯設定是不是出了什麼錯誤。

**Reply by xiedeacc** [reply]
> https://nodejs.org/zh-cn/download/current/ 絕對是支援，就是不知道你的跨編譯設定是不是出了什麼錯誤

应该是openwrt的交叉编译工具问题
